components/access-list/ui/Container.js

```diff
@@ -0,0 +1,66 @@
+import styled, { css } from 'styled-components'
+import PropTypes from 'prop-types'
+
+import { LIST_VARIANTS } from './constants'
+import Member from './Member'
+
+const Container = styled.div`
+  display: grid;
+  align-items: center;
+  justify-items: center;
+  grid-template-columns: repeat(var(--member-row-count, 5), 1fr);
+  margin: 0;
+
+  ${({ theme, variant }) =>
+    variant === 'primary'
```

Comment:
Same here as in the other PRs not sure if it is just me with multiline ternaries

Reply:
Since variant can only be one of two values, it works as a ternary here. (I could have used a bool, but a variant is typically a string so I stuck with that.) As for ternaries in React generally, I prefer them over the `&&` conditional as they are less prone to `falsy` surprises and I don’t have to add the `=> { if () {} else {} }`.

https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx